### ISSUE TITLE
Rule Change: Allow no-empty-blocks to ignore comment-only blocks

### ISSUE THREAD

**Issue by Pixel998** [eslint/css#477]
### What rule do you want to change?

no-empty-blocks

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```css
@media (min-width: 1200px) {
  .sidebar { /* Fix this later */ }
}
```

### What does the rule currently do for this code?

The rule reports the block as empty.

The rule currently treats comments as not being content, so a block containing only a comment is reported. There is no option to change this behavior.

### What will the rule do after it's changed?

With a new option, users could choose for comment-only blocks to be treated as non-empty, so the example above would not be reported.

### Participation

- [x] I am willing to submit a pull request to implement this change.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

This would bring `no-empty-blocks` closer to Stylelint’s `block-no-empty`, which does not report comment-only blocks by default: https://stylelint.io/user-guide/rules/block-no-empty/

**Comment by DMartens** [comment]
I agree with the change to add an option for this to be consistent with the respective stylelint rule.
For clarity should blocks with only a comment be allowed or disallowed by default?

**Comment by Pixel998** [comment]
I think we should disallow comment-only blocks by default (keeping the current behavior).

**Comment by Tanujkanti4441** [comment]
Idea sounds good to me. We can name it something like `allowCommentOnlyBlocks`?
And how about if the rule provides a suggestion as well to add a comment inside the empty block, only if this option is enabled?

**Comment by nzakas** [comment]
For comparison, the ESLint [`no-empty`](https://eslint.org/docs/latest/rules/no-empty) rule has always allowed comment-only blocks by default. I'd vote for having that also be the default behavior here with an `ignoreCommentOnlyBlocks` option to turn that behavior off if desired.

**Comment by Tanujkanti4441** [comment]
>For the purposes of this rule, comments do not count as content and this rule warns when the only content inside of a block is a comment.

I was skeptical about allowing comments by default because of this sentence in the rule docs.

But since it makes sense to me to make the rule's behavior similar `no-empty` core rule i think we can go with `ignoreCommentOnlyBlocks` option with default `true` and `no-empty` also provides the suggestion to add a comment inside blocks.
